examples/builder/vsphere-iso/alpine/alpine-3.20.3.pkr.hcl

```diff
@@ -12,20 +21,36 @@ source "vsphere-iso" "autogenerated_1" {
   CPUs                 = 1
   RAM                  = 512
   RAM_reserve_all      = true
-  boot_command         = ["root<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "setup-alpine -f /media/floppy/answerfile<enter>", "<wait5>", "jetbrains<enter>", "jetbrains<enter>", "<wait5>", "y<enter>", "<wait10><wait10><wait10><wait10>", "reboot<enter>", "<wait10><wait10>", "root<enter>", "jetbrains<enter><wait>", "mount -t vfat /dev/fd0 /media/floppy<enter><wait>", "/media/floppy/SETUP.SH<enter>"]
+  boot_command = ["root<enter><wait>",
+    "mount -t vfat /dev/fd0 /media/floppy<enter><wait>",
+    "setup-alpine -f /media/floppy/answerfile<enter>",
+    "<wait5>",
+    "VMw@re1!<enter>",
+    "VMw@re1!<enter>",
+    "<wait5>",
+    "y<enter>",
+    "<wait10><wait10><wait10><wait10><wait10><wait10>",
+    "reboot<enter>",
+    "<wait10><wait10>",
+    "root<enter>",
+    "VMw@re1!<enter><wait>",
+    "mount -t vfat /dev/fd0 /media/floppy<enter><wait>",
+    "/media/floppy/SETUP.SH<enter>"
+  ]
```

Comment:
You could also update this to use the following to avoid repeating the password in the boot command.

```hcl
variable "root_password" {
  type    = string
  default = "VMw@re1!"
}

source "vsphere-iso" "example" {
  // other configurations...
  boot_command = [
    "root<enter><wait>",
    "mount -t vfat /dev/fd0 /media/floppy<enter><wait>",
    "setup-alpine -f /media/floppy/answerfile<enter>",
    "<wait5>",
    "${var.root_password}<enter>",
    "${var.root_password}<enter>",
    "<wait5>",
    "y<enter>",
    "<wait10><wait10><wait10><wait10><wait10><wait10>",
    "reboot<enter>",
    "<wait10><wait10>",
    "root<enter>",
    "${var.root_password}<enter><wait>",
    "mount -t vfat /dev/fd0 /media/floppy<enter><wait>",
    "/media/floppy/SETUP.SH<enter>"
  ]
}
```